core/contracts/financial-templates/implementation/FeePayer.sol

```diff
@@ -67,7 +87,9 @@ contract FeePayer is Testable {
             collateralCurrency.safeTransfer(msg.sender, latePenalty.rawValue);
         }
 
-        return regularFee.add(latePenalty);
+        totalPaid = regularFee.add(latePenalty);
+        FixedPoint.Unsigned memory effectiveFee = totalPaid.divCeil(_pfc);
```

Comment:
We should probably move the rounding error tests from `PricelessPositionManager.js` into a `FeePayer` test.

Reply:
While that sounds reasonable, the FeePayer contract can't be instantiated on its own, so we would need to use an existing (or make a new) contract to test it. WDYT?

Are you suggesting any new tests or just moving existing ones? If just moving existing ones, I'd prefer to put that in a follow-up PR to distinguish them from functional changes.

Reply:
Not suggesting new tests since you didn't add new arithmetic in the `FeePayer` logic. I just wanted to make sure that `Liquidatable`'s regular fees are well tested, but since its calling the same methods as `PricelessPositionManager` then its already tested